Basket-Donation-Payments/Basket-Donation-Payments/Models/APIManager.m

```diff
@@ -25,35 +30,44 @@ -(void)fetchBraintreeClientToken {
     NSURL *clientTokenURL = [NSURL URLWithString:@"https://braintree-sample-merchant.herokuapp.com/client_token"];
     NSMutableURLRequest *clientTokenRequest = [NSMutableURLRequest requestWithURL:clientTokenURL];
     [clientTokenRequest setValue:@"text/plain" forHTTPHeaderField:@"Accept"];
-
+    
     [[[NSURLSession sharedSession] dataTaskWithRequest:clientTokenRequest completionHandler:^(NSData * _Nullable data, NSURLResponse * _Nullable response, NSError * _Nullable error) {
         // TODO: Handle errors
         //NSString *clientToken = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
         NSString *clientToken = @"CLIENT_TOKEN_FROM_SERVER"; //TODO: demo client token
-
+        
```

Comment:
nit: there's a setting in Xcode to delete trailing whitespaces automatically:
![Screen Shot 2020-07-28 at 1 32 12 PM](https://user-images.githubusercontent.com/1676646/88718656-f75f0600-d0d6-11ea-9f11-47198b8eccf4.png)

Reply:
Thank you!